regression_tests/stages/run_demodjango_smoke_tests.sh

```diff
@@ -7,5 +7,24 @@ cd "${CODEBUILD_SRC_DIR}/demodjango"
 echo -e "\nInstall dependencies"
 poetry install
 
+echo -e "\nAssume platform-sandbox role to trigger environment pipeline"
+assumed_role=$(aws sts assume-role \
+    --role-arn "arn:aws:iam::$PLATFORM_SANDBOX_AWS_ACCOUNT_ID:role/regression-tests-assume-role-for-platform-tools" \
+    --role-session-name "pull-request-regression-tests-$(date +%s)")
+PLATFORM_SANDBOX_AWS_ACCESS_KEY_ID=$(echo $assumed_role | jq -r .Credentials.AccessKeyId)
+PLATFORM_SANDBOX_AWS_SECRET_ACCESS_KEY=$(echo $assumed_role | jq -r .Credentials.SecretAccessKey)
+PLATFORM_SANDBOX_AWS_SESSION_TOKEN=$(echo $assumed_role | jq -r .Credentials.SessionToken)
+
+export AWS_ACCESS_KEY_ID=$PLATFORM_SANDBOX_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$PLATFORM_SANDBOX_AWS_SECRET_ACCESS_KEY
+export AWS_SESSION_TOKEN=$PLATFORM_SANDBOX_AWS_SESSION_TOKEN
+
+echo -e "\nFetch parameters from AWS Parameter Store"
+USERNAME=$(aws ssm get-parameter --name "/copilot/demodjango/${TARGET_ENVIRONMENT}/secrets/BASIC_AUTH_USERNAME" --with-decryption --query "Parameter.Value" --output text)
+PASSWORD=$(aws ssm get-parameter --name "/copilot/demodjango/${TARGET_ENVIRONMENT}/secrets/BASIC_AUTH_PASSWORD" --with-decryption --query "Parameter.Value" --output text)
```

Comment:
Yeah that would make more sense. I think we'd have to use the arn rather than the name of the param to grab it from another account. Also I'm not sure the target environment matters. Right now I've set the same basic auth creds for each env. I should probs just create one param and reference that everywhere.

I thought Lyndon's work here was relevant, but not sure it is https://github.com/uktrade/platform-tools/pull/531

Reply:
Actually I'm not sure it's possible because those live in another account, so I think we'd still need the assume role logic. We could put that logic in the build phase of `buildspec.test.yml` but I'm not sure that's an improvement on what I've got now. Just moves the mess somewhere else. Also not all the other regression tests need them and perhaps at some point we'll want to give the option of selecting which tests to run instead of the whole suite

Reply:
Hmm. Yes, probably would get messy. Not sure how or if you could do cross-account param store lookup in the codebuild terraform even with the appropriate IAM config.